plugins/gfm/src/main/kotlin/org/jetbrains/dokka/gfm/renderer/CommonmarkRenderer.kt

```diff
@@ -259,25 +262,26 @@ open class CommonmarkRenderer(
         distinct.values.forEach { entry ->
             val (instance, sourceSets) = entry.getInstanceAndSourceSets()
 
+            buildParagraph()
             buildSourceSetTags(sourceSets)
+            buildNewLine()
 
             instance.before?.let {
-                buildNewLine()
-                append("Brief description")
-                buildNewLine()
+                ///append("Brief description")
+                ///buildNewLine()
```

Comment:
Please remove the comments here and below, if they are not needed

Reply:
Removed.